plugins/destination/clickhouse/client/migrate.go

```diff
@@ -123,6 +123,11 @@ func (c *Client) dropTable(ctx context.Context, table *schema.Table) error {
 }
 
 func needsTableDrop(change schema.TableColumnChange) bool {
+	// Support for adding the cq_client_id column without dropping the table
+	if change.Type == schema.TableColumnChangeTypeAdd && change.Current.Name == schema.CqClientIDColumn.Name {
+		return false
```

Comment:
Is there any way at this point to know if `order_by` was configured explicitly? If I understand the PR description correctly, it's either configured and we can add the column, or it's not, and we need to drop the table?

Reply:
> and we need to drop the table?

If there's an `order_by` mismatch it will fail a bit later (regardless of the `_cq_client_id` change) with a proper error message since `order_by` changes also require a full drop.

https://github.com/cloudquery/cloudquery/blob/c44d7f10658c6c98fb1c5306859efee26a8d27dd/plugins/destination/clickhouse/client/migrate.go#L209